packages/near-membrane-base/src/membrane.ts

```diff
@@ -217,6 +225,26 @@ export function init(
         };
     }
 
+    // This wrapping mechanism provides the means to add instrumentation
+    // to the callable functions used to coordinate work between the sides
+    // of the membrane.
+    // TODO: do we need to pass more info into instrumentation hooks?
+    function instrumentCallableWrapper<T extends (...args: any[]) => any>(fn: T, label: string): T {
+        if (instrumentation) {
```

Comment:
I'm keeping the condition inside the function, but it might very well be use to declare the `instrumentCallableWrapper` conditionally, and eliminate the `if` condition from inside the wrapper function.

Reply:
define `instrumentCallableWrapper` conditionally seems to have less overhead during runtime.

Reply:
Probably does not matter much as it onlyu impacts the initialization?